docs/event-dispatcher.md

```diff
@@ -76,8 +85,46 @@ Example:
       "microblock_sequence": "3",
       "microblock_hash": "0x9304fcbcc6daf5ac3f264522e0df50eddb5be85df6ee8a9fc2384c54274daaac",
       "microblock_parent_hash": "0x4893ab44636023efa08374033428e44eca490582bd39a6e61f3b6cf749b474bd"
+    },
+    {
+      "burnchain_op": {
+        "TransferStx": {
+          "block_height": 10,
+          "burn_header_hash": "1410131010105914101010101013704010101010221010101010101010101010",
+          "memo": "0x000102030405",
+          "recipient": {
+            "address": "SP24ZBZ8ZE6F48JE9G3F3HRTG9FK7E2H6K2QZ3Q1K",
+            "address_hash_bytes": "0x89f5fd1f719e4449c980de38e3504be6770a2698",
+            "address_version": 22
+          },
+          "sender": {
+            "address": "ST2QKZ4FKHAH1NQKYKYAYZPY440FEPK7GZ1R5HBP2",
+            "address_hash_bytes": "0xaf3f91f38aa21ade7e9f95efdbc4201eeb4cf0f8",
+            "address_version": 26
+          },
+          "transfered_ustx": 10,
+          "txid": "85aa2106186723f3c4f1d8bb58e3a02746ca9be1be9f4be0c6557079e1f660e6",
```

Comment:
Is this burnchain/bitcoin txid or Stacks txid?

Reply:
it's the burnchain txid